### ISSUE TITLE
No source code?

### ISSUE THREAD

**Issue by philstopford** [xoblite/xoblite-shell#1]
The readme suggests this repository holds the source code, but only the readme appears to exist. Is the source on-the-way?

**Comment by Aptronymist** [comment]
I have to agree, is the source code going to be here at some point? It's been 3 years.

**Comment by xoblite** [comment]
> [added *.cpp/h 2025-04-14 qwilk]
> - At long last, the xoblite source code documentation is now available at [http://xoblite.net/shell/?SourceCode](http://xoblite.net/shell/?SourceCode)
> -> You can use this tool together with the file references in the [Changes](https://xoblite.net/shell/changes_bb5.txt) file to look into the specifics of each change, and more! :D

BR//Karl aka qwilk/@xoblite
